client/src/views/student/MyClasses.vue

```diff
@@ -63,22 +63,23 @@ export default {
         .then(async () => {
           this.fullscreenLoading = true;
 
-          let data = await this.cancelRegisteredClass(row.ClassID);
+          let data = await this.cancelRegisteredClass({
+            classId: row.ClassID
+          });
 
-          if (data) {
-            if (data.success) {
-              this.status = false;
-              Message.success('Unenroll successfully!');
-            } else {
-              Message.error(data.msg);
-            }
+          if (!data) {
+            Message.error(data.msg);
+          } else if (data) {
+            this.status = false;
+            Message.success('Unenroll successfully!');
           }
-          await this.getMyClasses();
 
+          await this.getMyClasses();
```

Comment:
`getMyClasses` ở đây làm gì ?

Reply:
Get Myclasses ở đây để nó cập nhật lại list classes mới sau khi unenroll ạ

Reply:
```js
async function getMyClasses() {
  try {
    let respone = await axios.get(`${process.env.VUE_APP_API_BACKEND}/account/me/classes`, {
      headers: authHeader()
    });
    return respone.data.classes;
  } catch (error) {
    throw error;
  }
}
```
phần này hoàn toàn chỉ đọc thôi chứ có ghi ở đâu đâu nhỉ mà update ?

Reply:
Nó fetch data về và đổ vào state của table, khi quay tròn xong thì nó hiện ra data mới ạ